appinfo/info.xml

```diff
@@ -3,14 +3,13 @@
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>calendar</id>
 	<name>Calendar</name>
-	<summary>Calendar GUI for Nextcloud's CalDAV server</summary>
-	<description><![CDATA[A calendar app for Nextcloud. Easily sync events from various devices with your Nextcloud and edit them online.
+	<summary>A Calendar app for Nextcloud</summary>
```

Comment:
Nextcloud should stay in the summary, because I have no better wording, and I see no problem, as other apps do this the same way 😉 ... Description was changed 💪